pkg/tests/upgrade_tests/shapshot.go

```diff
@@ -164,42 +164,55 @@ func (s dbSnapshot) writeToFile(f *os.File) error {
 	return w.Flush()
 }
 
-func printOutputDiff(t *testing.T, upgradedOuput string, pristineOutput string, msg string) bool {
-	if upgradedOuput == pristineOutput {
+func printOutputDiff(t *testing.T, a string, b string, msg string) bool {
+	if a == b {
 		return false
 	}
 	dmp := diffmatchpatch.New()
-	diffs := dmp.DiffMain(upgradedOuput, pristineOutput, false)
+	diffs := dmp.DiffMain(a, b, false)
 	t.Logf("%s:\n%v", msg, dmp.DiffPrettyText(diffs))
 	return true
 }
 
-func PrintDbSnapshotDifferences(t *testing.T, pristineDbInfo dbSnapshot, upgradedDbInfo dbSnapshot) {
+func PrintDbSnapshotDifferences(t *testing.T, a dbSnapshot, b dbSnapshot) {
 	t.Errorf("upgrade differences")
-	printOutputDiff(t, fmt.Sprintf("%+v", upgradedDbInfo.schemaNames), fmt.Sprintf("%+v", pristineDbInfo.schemaNames), "schema names differ")
-	printOutputDiff(t, upgradedDbInfo.users, pristineDbInfo.users, "users differ")
-	printOutputDiff(t, upgradedDbInfo.extensions, pristineDbInfo.extensions, "extension outputs differ")
-	printOutputDiff(t, upgradedDbInfo.promscale, pristineDbInfo.promscale, "promscale extension members differ")
-	printOutputDiff(t, upgradedDbInfo.schemaOutputs, pristineDbInfo.schemaOutputs, "schema outputs differ")
-	printOutputDiff(t, upgradedDbInfo.defaultPrivileges, pristineDbInfo.defaultPrivileges, "default privileges differ")
+	printOutputDiff(t, fmt.Sprintf("%+v", a.schemaNames), fmt.Sprintf("%+v", b.schemaNames), "schema names differ")
```

Comment:
Would it make sense to rename those variables to `before` and `after` so we don't use short names across the context of a whole function that spans ~40 lines?